packages/mgt-chat/src/components/ManageChatMembers/ListChatMembers.tsx

```diff
@@ -21,7 +21,7 @@ interface ListChatMembersProps {
   closeParentPopover: () => void;
 }
 
-const RemovePerson = bundleIcon(Dismiss24Regular, () => null);
+const RemovePerson = bundleIcon(Dismiss24Filled, Dismiss24Regular);
```

Comment:
We can change this to make the dismiss icon only show on hover like this:
```suggestion
const RemovePerson = bundleIcon(Dismiss24Filled, () => <></>);
```